make_helpers/build_macros.mk

```diff
@@ -401,9 +406,18 @@ else
 	       const char version_string[] = "${VERSION_STRING}";' | \
 		$$(CC) $$(TF_CFLAGS) $$(CFLAGS) -xc -c - -o $(BUILD_DIR)/build_message.o
 endif
+ifneq ($(findstring armlink,$(notdir $(LD))),)
+	$$(Q)$$(LD) -o $$@ $$(TF_LDFLAGS) $$(LDFLAGS) --entry=bl${1}_entrypoint \
+		--predefine="-D__LINKER__=$(__LINKER__)" \
+		--predefine="-DTF_CFLAGS=$(TF_CFLAGS)" \
+		--map --list="$(MAPFILE)" --scatter=$(BL31_SCATTERFILE) \
```

Comment:
Since this macro is common for all BL images, suggest to modify the `--scatter` option as:
--scatter=${PLAT_DIR}/scat/bl${1}.scat

Which means, the current tegra.scat has to be renamed to bl31.scat and placed in the appropriate folder.

Reply:
Done